package.json

```diff
@@ -22,7 +22,7 @@
     "@uniswap/sdk-core": "3.0.1",
     "@uniswap/v3-sdk": "3.6.3",
     "@walletconnect/web3-provider": "^1.5.4",
-    "antd": "4.16.13",
+    "antd": "4.18.0",
```

Comment:
**Question**

Reply:
Thanks for the review! The antd update brings in the change to the input form to use showCount. It was added in 4.7.x.